shared/locales/en/site.json

```diff
@@ -131,6 +131,14 @@
         "title": "Connection May Be Insecure",
         "note": "Shown as the button text when the connection is using an invalid certificate."
     },
+    "phishingWebsite": {
+        "title": "Site May Be Deceptive",
+        "note": "Shown as the button text when the URL is reported as a phishing website."
+    },
+    "phishingWebsiteDesc": {
```

Comment:
Are these strings confirmed yet?

Reply:
Yes, according this https://app.asana.com/0/38424471409662/1207197456211299/f but they might get changed during Ship Review